transformers/component_tests/queries/test_helpers/test_helpers.go

```diff
@@ -488,11 +489,12 @@ func FlapBidFromValues(bidId, dealt, updated, created string, bidValues map[stri
 
 func FlipBidFromValues(bidId, ilkId, urnId, dealt, updated, created string, bidValues map[string]interface{}) FlipBid {
 	return FlipBid{
-		commonBid: commonBidFromValues(bidId, dealt, updated, created, bidValues),
-		IlkId:     ilkId,
-		UrnId:     urnId,
-		Gal:       bidValues[storage.BidGal].(string),
-		Tab:       bidValues[storage.BidTab].(string),
+		commonBid:   commonBidFromValues(bidId, dealt, updated, created, bidValues),
+		IlkId:       ilkId,
+		UrnId:       urnId,
+		Gal:         bidValues[storage.BidGal].(string),
+		Tab:         bidValues[storage.BidTab].(string),
+		FlipAddress: fakes.FakeAddress.Hex(),
```

Comment:
Does the fact that we're using this value here mean that we need to use the same on when we're setting up/asserting against flip records in the tests calling this function? Wondering if we might want to add a param to inject it to avoid coupling to this specific string

Reply:
I like the idea of passing in the address here, and think I was avoiding doing that earlier because this method is used in LOTS of places! I had been setting the FlipAddress on the returned FlipBid for tests where it matters, but I think that passing in the address is definitely the cleaner way to handle it.

Reply:
Cool, no worries from my perspective if we want to make that a separate story, if it would blow up the diff

Reply:
I think I was being dramatic, it wasn't _that_ many places 😜